src/komponenter/motestotte/motestotte.tsx

```diff
@@ -4,81 +4,94 @@ import { Systemtittel, Normaltekst } from 'nav-frontend-typografi';
+    const { erSykmeldtMedArbeidsgiver } = React.useContext(BrukerInfoContext).data;
+    const { systemtittel, tekster } = hentTekster(erSykmeldtMedArbeidsgiver);
     const opprettetRegistreringDatoString = selectOpprettetRegistreringDato(data);
-    const opprettetRegistreringDato = opprettetRegistreringDatoString ? new Date(opprettetRegistreringDatoString) : null;
+    const opprettetRegistreringDato = opprettetRegistreringDatoString
+        ? new Date(opprettetRegistreringDatoString)
+        : null;
     const foreslattInnsatsgruppe = selectForeslattInnsatsgruppe(data)!; // Komponent blir rendret kun hvis foreslått innsatsgruppe er satt
-    const antallTimer = antallTimerSidenRegistrering(opprettetRegistreringDato!)
+    const antallTimer = antallTimerSidenRegistrering(opprettetRegistreringDato!);
+    const dinSituasjon = selectDinSituasjonSvar(data) || 'INGEN_VERDI';
+
+    const harGyldigMotestottebesvarelse = (): boolean => {
+        if (!opprettetRegistreringDato || !motestotteData) return false;
+        return opprettetRegistreringDato <= new Date(motestotteData.dato);
+    };
+
+    const skalViseMotestotteLenke =
+        dinSituasjon !== 'ER_PERMITTERT' &&
+        oppfolgingData.servicegruppe === Servicegruppe.BKART &&
+        !harGyldigMotestottebesvarelse() &&
+        opprettetRegistreringDato !== null &&
+        opprettetRegistreringDato >= LANSERINGSDATO_MOTESTOTTE &&
+        !oppfolgingData.reservasjonKRR &&
+        foreslattInnsatsgruppe === ForeslattInnsatsgruppe.BEHOV_FOR_ARBEIDSEVNEVURDERING;
 
     React.useEffect(() => {
         uniLogger('motestotte.visning', { antallTimer, foreslattInnsatsgruppe });
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, []);
 
-    const handleButtonClick = () => {
-        uniLogger('motestotte.gatil', { antallTimer, foreslattInnsatsgruppe });
-        window.location.href = motestotteLenke;
-    };
-
-    if (erSykmeldtMedArbeidsgiver) {
-        return (
-            <Panel border className="ramme blokk-s">
-        <section className="motestotte blokk-m">
-            <div className="innhold">
-                <Systemtittel tag="h1" className="blokk-xs">
-                    Du kan få mer veiledning
-                </Systemtittel>
-                <Normaltekst className="blokk-m motestotte__tekst">
-                    Du har svart at du trenger mer veiledning nå som retten til sykepenger nærmer seg
-                    slutten. </Normaltekst>
-                <Normaltekst className="blokk-m motestotte__tekst">
-                    Vi ønsker å bli bedre kjent med situasjonen din, slik at du kan få veiledning som passer for
-                    deg.
-                </Normaltekst>
-                <Hovedknapp onClick={handleButtonClick}>
-                    Start
-                </Hovedknapp>
-            </div>
-        </section>
-        </Panel>
-        )
-    }
+    if (skalViseMotestotteLenke) return null;
```

Comment:
skal ikke denne være `if (!skalViseMotestotteLenke) return null`? Eller har jeg gått meg vill i logikken?

Reply:
🥇 Godt obeservert. Det var jeg som hadde tulla for å teste manuelt. Fanges opp av testene nå